### PR TITLE
Fix minor bugs in the User Guide

### PR DESCRIPTION
**Pull request by JunWei3112**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -362,7 +359,6 @@ Format: `add -l n/LESSON_NAME [c/LESSON_CAPACITY] [p/LESSON_PRICE] [t/LESSON_TIM
 
 * The lesson's capacity, price and timing are optional details for you to include.
 * The lesson name should only contain alphanumeric characters and spaces.
-* Duplicate lessons cannot be added. A lesson counts as a duplicate if it has the same lesson name as an existing lesson. Letter case is ignored when determining if two lesson names are the same.
```
Comment (wpinrui) [inline docs/UserGuide.md:365]: Sorry if this is a blur qn but why did u remove this haha

File: docs/UserGuide.md [hunk 2]
```diff
@@ -445,35 +441,35 @@ Examples:
 * `list` displays all students and lessons in TutorAid by only showing their names and list indexes.
 * `list -a` displays all students and lessons in TutorAid while showing all of their fields' data.
 
-### Adding students to lessons: `add -sl`
+### Adding student to lesson: `add -sl`
```
Comment (wpinrui) [inline docs/UserGuide.md:444]: Since this is for 2101... Adding a student to a lesson? 😆

File: docs/UserGuide.md [hunk 3]
```diff
@@ -445,35 +441,35 @@ Examples:
...
 {:.no_toc}
-Adds students to lessons.
+Adds a student to a lesson.
```
Comment (wpinrui) [inline docs/UserGuide.md:446]: Then maybe here can be "Tells TutorAid that a student attends a particular lesson"

**Review by wpinrui — COMMENTED** [review]
Small nitpicks, otherwise LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (JunWei3112) [reply docs/UserGuide.md:365]: Oh i removed it as this is already mentioned in lines 369 and 370

File: docs/UserGuide.md (hunk 2 above)
Comment (JunWei3112) [reply docs/UserGuide.md]: Fixed, thanks! 😂 Updated the command description for `del -sl` too

File: docs/UserGuide.md (hunk 3 above)
Comment (JunWei3112) [reply docs/UserGuide.md]: Updated it, thanks!

**Review by wpinrui — APPROVED** [review]
LGTM

**Review by Preshita01 — APPROVED** [review]
LGTM

**Pull request merged**